queue_services/entity-filer/src/entity_filer/filing_processors/incorporation_filing.py

```diff
@@ -80,6 +80,7 @@ def _update_cooperative(incorp_filing: Dict, business: Business, filing: Filing)
         document = Document()
         document.type = DocumentType.COOP_RULES.value
         document.file_key = rules_file_key
+        document.file_name = cooperative_obj.get('rulesFileName', 'rules.pdf')
```

Comment:
I've included fallbacks here, doing fallbacks for `dpath.util.get(filing, '/alteration/memorandumFileName')` doesn't seem too easy